Vagrantfile

```diff
@@ -104,4 +104,23 @@ Vagrant.configure("2") do |config|
       virtualbox.customize ["setextradata", :id, "VBoxInternal2/EfiGopMode", "4"]
     end
   end
+
+  config.vm.define "solaris" do |node|
+    node.vm.box =  "jonatasbaldin/solaris11"
```

Comment:
you might give this 11.4 box a try: https://app.vagrantup.com/MartijnDwars/boxes/solaris11_4